src/markdown-pages/automate-workflows/get-started-terraform.mdx

```diff
@@ -16,139 +16,210 @@ resources:
 
 <Steps>
+
+<!-- TERRAFORM CONFIGURATION -->
+
 <Step>
 
-## Bootstrap your provider configuration
+## Bootstrap Terraform and the New Relic provider
 
-Our Terraform Provider detects the environment variables above when running Terraform commands.
+Start by initializing a working directory and creating a Terraform configuration file:
 
-You can set the environment variables in your `.bash_profile` or `.bashrc` file.
+```bash
+mkdir terraform-project && cd terraform-project
+touch main.tf
+```
 
-Set the following environment variables:
+Next, instruct Terraform to install and use the New Relic provider, by setting the `terraform` and `required_providers` blocks in **main.tf**:
 
-- Set your [New Relic Personal API key](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#personal-api-key) with the `NEW_RELIC_API_KEY` environment variable. Most Personal API keys begin with the prefix `NRAK-`.
-- Set your [New Relic Admin API key](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#admin) with the `NEW_RELIC_ADMIN_API_KEY` environment variable. Most Admin API keys begin with the prefix `NRAA-`.
-- Set your [New Relic Account ID](https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/account-id) with the `NEW_RELIC_ACCOUNT_ID` environment variable.
-- Set your New Relic [region](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account) with the `NEW_RELIC_REGION` environment variable. Your region is `US` if your account settings page is located at `rpm.newrelic.com`, and `EU` if your account is located at `rpm.eu.newrelic.com`.
+```hcl
+terraform {
+  # Require Terraform version 0.13.x (recommended)
+  required_version = "~> 0.13.0"
 
-```bash lineNumbers=false
-# Add this to your .bash_profile
-export NEW_RELIC_API_KEY=NRAK-...
-export NEW_RELIC_ADMIN_API_KEY=NRAA-...
-export NEW_RELIC_ACCOUNT_ID=12345
-export NEW_RELIC_REGION=US
-# Or set it inline when running `terraform plan` or `terraform apply`:
-$ NEW_RELIC_API_KEY=NRAK-... terraform apply
+  # Require version 2.11.x of the New Relic provider
+  required_providers {
+    newrelic = {
+      source  = "newrelic/newrelic"
+      version = "~> 2.11.0"
+    }
+  }
+}
 ```
 
+<Tip>
+
+We recommend that you pin your New Relic provider version by setting a [version constraint](https://www.terraform.io/docs/configuration/version-constraints.html) similar to the example (`"~> 2.11.0"`). Using the right version constraint for your setup can provide better stability with your Terraform runs.
+
+While this example uses version 2.11.x of the New Relic provider, feel free to specify the latest provider version.
```

Comment:
> feel free to specify the latest provider version.

Is this actually ok? Couldn't a later version introduce breaking changes?

Reply:
Good call. But I wouldn't want a customer starting on an outdated version. In a month's time we might be at version 2.28.6 or something. Theoretically as long as it's a 2.x version, it should be okay since we strive to stick to semver and we bump major versions if a breaking change is introduced (in this case the next major would 3.x). Maybe we reword it?

Reply:
Could you just pin it using `~> 2.11`? That would be the same as `>= 2.11, < 3.0`, right?

Reply:
Oh right, there's a zero in the example. I think I'm just used to specifying a version constraint that will only update the patch version. I'll update the constraint.